aten/src/ATen/core/op_registration/infer_schema.h

```diff
@@ -22,6 +22,8 @@ namespace infer_schema {
 struct ArgumentDef final {
   using GetTypeFn = TypePtr();
   GetTypeFn* getTypeFn;
+  ArgumentDef() = default;
```

Comment:
do we actually need a default constructor? if so, I'd set getTypeFn to be nullptr explicitly

Reply:
This is only needed by MSVC. Without this, we can not create an `std::array` of `ArgumentDef`

Reply:
I have update this line to set getTypeFn to nullptr